ffi/system-command/win32ui.dart

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:ffi' as ffi;
```

Comment:
We don't really use `as ffi` in any of the samples inside the SDK anymore.

Reply:
dropped the `as ffi`.